audit/get-org-mem-usage.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+##
+# Get the memory quota and usage for a specific organization.
+# 
+# Example usage:
+# ~$ ./get-org-mem-usage.sh sandbox-gsa
+
+# Org name: sandbox-gsa
+# GUID: 70a1c8e4-6555-44ff-9ed0-57711c90104f
+# Quota (GB): 50
+# Usage (GB): 8
+#
+## 
+
+if [[ ! $(which jq) ]]; then
+  echo "Error: You must have the jq utility installed to run this script. https://stedolan.github.io/jq/" >&2
+  exit 1
+fi
+
+if [ -z "$1" ]
+  then
+    echo "You must supply an org name"
+    exit 1
+fi
+
+# Get organization details
+org=$(cf curl v3/organizations/?names=$1)
+
+# Get the guid for the organization
+guid=$(echo $org | jq '.resources[] | .guid' | sed 's/\"//g')
```

Comment:
You can use the `-j` flag with jq to remove the quotes

Reply:
Good call, @apburnes. Change made.